### PR TITLE
#162390805 Implement all UI feedbacks

### PR DESCRIPTION
**Pull request by shaolinmkz**
#### What does this PR do?
Have all feedback from LFA implemented

#### Description of Task to be completed?
Having the following bugs below fixed:
* Removed the `iFrame` tag
* Improved the uniformity of the report form
* Added HTML5 validation
* Added a dummy login
* Made footer height smaller
* Distinguish `button` from `anchor` tags
* Centralized profile name, match avatar horizontal centre
* Pushed `footer` down by some margins
* Increase the line height of the user profile details and aligned it to the left
* Made the `modify` button more descriptive

#### What are the relevant pivotal tracker stories?
[#162390805](https://www.pivotaltracker.com/story/show/162390805)

#### Screenshots
views

### CODE REVIEW

File: ui/html/home.html [hunk 1]
```diff
@@ -68,12 +122,12 @@ <h1>SEE SOMETHING SAY SOMETHING...</h1>
...
-            <p>&copy; <span>i</span>Reporter 2018</p>
+            <p>&copy; <span class="theme-orange">i</span>Reporter 2018</p>
         </div>
 
     </footer>
     <script src="../scripts/main.js"></script>
+    <script src="../scripts/injectmarkups.js"></script>
 </body>
 
 </html>
```
Comment (otseobande) [inline ui/html/home.html]: Please end your files with a new line.

File: ui/scripts/injectmarkups.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+/**
+ * Indicates Current records switch
+ * @param {object} event - event object
+ */
+const injectIncidentRecord = (event) => {
+    let displayPost = document.getElementsByClassName("post-display")[0];
+    if (event.target.id === "redflag-record") {
+        displayPost.innerHTML = "";
+        displayPost.innerHTML =`
```
Comment (otseobande) [inline ui/scripts/injectmarkups.js]: Good use of template strings

File: ui/scripts/main.js [hunk 3]
```diff
@@ -148,10 +150,10 @@ window.addEventListener("click", (e) => {
  * @param {object} event - event object
  */
```
Comment (otseobande) [inline ui/scripts/main.js:160]: Please specify `@return` or `@returns` in your JSDoc.

File: ui/scripts/main.js [hunk 4]
```diff
@@ -253,11 +219,11 @@ const findMe = (e) => {
 	} else if (navigator.geolocation) {
 		navigator.geolocation.getCurrentPosition(revealCoordinates, errorResponse);
 		loading.style.display = "inline-block";
```
Comment (otseobande) [inline ui/scripts/main.js:218]: Please use an indentation of 2 spaces for this file.

**Review by otseobande — CHANGES_REQUESTED** [review]
You've done well implementing the feedback given to you so far. I left some comments I need you to address.

File: ui/scripts/injectmarkups.js (hunk 2 above)
Comment (shaolinmkz) [reply ui/scripts/injectmarkups.js]: Thank you....

File: ui/scripts/main.js (hunk 3 above)
Comment (shaolinmkz) [reply ui/scripts/main.js:160]: Okay

File: ui/scripts/main.js (hunk 4 above)
Comment (shaolinmkz) [reply ui/scripts/main.js:218]: Okay

File: ui/scripts/main.js [hunk 5]
```diff
@@ -112,11 +121,11 @@ const controlHamburger = (event) => {
 };
 
 
-
 window.addEventListener("click", controlHamburger);
 
 /**
  * Hamburger menu initializer
+ * @return {undefined}
```
Comment (otseobande) [inline ui/scripts/main.js]: You don't need to write jsdoc for builtin functions or callbacks

File: ui/scripts/main.js [hunk 6]
```diff
@@ -334,24 +312,128 @@ const getAddress = (event) => {
...
-	google.maps.event.addDomListener(window, 'load', initialize);
+	window.addEventListener("load", initAutocomplete);
+}
+
+
+/**
+ * Edit comment
+ * @return {undefined}
+ */
+const editComment = (event) => {
```
Comment (otseobande) [inline ui/scripts/main.js:334]: Please document the argument to this function

File: ui/scripts/main.js [hunk 7]
```diff
@@ -334,24 +312,128 @@ const getAddress = (event) => {
...
+
+		//insert the string of mark-up into the HTML element with class="insert-location-editing-tag-here"
+		target.children[14].innerHTML += `
+        <input type="text" class="location-input" value="${initialLoaction.innerHTML}" />
+        <span class="editing-button-tag">
+         <a class="red edit exit">cancel</a>
+         <span>&nbsp;</span><a class="blue edit">update</a></span> `
+	}
+
+
```
Comment (otseobande) [inline ui/scripts/main.js]: I don't think these extra lines are necessary

**Pull request merged**